src/signals/incident/components/IncidentForm/index.js

```diff
@@ -203,7 +205,10 @@ class IncidentForm extends React.Component {
         this.setIncident(formAction);
         next();
       } else {
-        window.scrollTo({ top: 0, behavior: 'smooth' });
+        const invalidControl = Object.values(this.form.controls).find(c => c.invalid);
+        const { name, values } = invalidControl.meta;
+        const valueSelector = !Array.isArray(values) && isObject(values) ? `-${Object.keys(values)[0]}1` : '';
```

Comment:
Feels error-prone to maintain this selector, although I can't think think of a different way to get the same functionality. 

Small improvement: looks like it is possible to remove `+ 1` at  `src/signals/incident/components/form/RadioInputGroup/RadioInputGroup.js#L45` so that you won't have to hardcode a `1` here.

Reply:
Agree that the solution is not ideal, but the best choice for now. Removing `+ 1` won't help. This 1 suffix is created by the reactive-forms framework.

Reply:
I'm quite sure the suffix is created by `idAttr={`${meta.name}-${key + 1}`}` in the file I linked. As far as I can see there is no use for it.